spec/docs/participant_declarations_spec.rb

```diff
@@ -2,17 +2,17 @@
 
 require "swagger_helper"
 
-RSpec.describe "Early Career Teacher Participation", type: :request, swagger_doc: "v1/api_spec.json" do
+RSpec.describe "Participant Declarations", type: :request, swagger_doc: "v1/api_spec.json" do
   let(:user) { create(:user) }
   let(:lead_provider) { create(:lead_provider) }
   let(:token) { LeadProviderApiToken.create_with_random_token!(lead_provider: lead_provider) }
   let(:bearer_token) { "Bearer #{token}" }
   let(:Authorization) { bearer_token }
 
-  path "/api/v1/early-career-teacher-participants" do
-    post "Add Early Career Teacher to Course" do
+  path "/api/v1/participant-declarations" do
+    post "Create course participant declarations" do
```

Comment:
This is ambiguous. "Create course ..... participant declarations"
```suggestion
    post "Create course-participant declarations" do
```